src/redux/uniswap.ts

```diff
@@ -177,17 +244,25 @@ export const uniswapUpdateFavorites = (
     : isArray(assetAddress)
     ? without(normalizedFavorites, ...assetAddress)
     : without(normalizedFavorites, assetAddress);
+  const updatedFavoritesMeta = await getUniswapFavoritesMetadata(
+    updatedFavorites
+  );
   dispatch({
-    payload: updatedFavorites,
+    payload: {
+      favorites: updatedFavorites,
+      favoritesMeta: updatedFavoritesMeta,
+    },
     type: UNISWAP_UPDATE_FAVORITES,
   });
   saveUniswapFavorites(updatedFavorites);
+  saveUniswapFavoritesMetadata(updatedFavoritesMeta);
 };
 
 // -- Reducer --------------------------------------------------------------- //
 
 export const INITIAL_UNISWAP_STATE: UniswapState = {
   favorites: DefaultUniswapFavorites['mainnet'],
+  favoritesMeta: DefaultUniswapFavoritesMeta['mainnet'],
```

Comment:
can import `Network` type and use `Network.mainnet` here and in line above on line 264

Reply:
can replace use of `networkTypes` with `Network` in this file